include/spdlog/details/os.h

```diff
@@ -128,6 +129,65 @@ constexpr inline unsigned short eol_size()
 }
 #endif
 
+//Return true and list if filename contains relative
+inline bool dir_check(const std::string& filename, std::list<std::string>& dirs)
+{
+#ifdef __linux__
+    std::string directory;
+    std::string::size_type index, previndex = 0, size;
+
+    index = filename.find("/", previndex);
+    if(!index)
```

Comment:
too complicated in my opinion. i am sure there are simpler ways to create set of folders without trying to create "/"